content/blog/deploy-stateful-mongodb-application-on-kubernetes-clusters-in-hpe-greenlake-for-containers-1.md

```diff
@@ -0,0 +1,547 @@
+        "t" : NumberLong(1)
+      },
+      "optimeDurable" : {
+        "ts" : Timestamp(1658309173, 1),
+        "t" : NumberLong(1)
+      },
+      "optimeDate" : ISODate("2022-07-20T09:26:13Z"),
+      "optimeDurableDate" : ISODate("2022-07-20T09:26:13Z"),
+      "lastHeartbeat" : ISODate("2022-07-20T09:26:18.007Z"),
+      "lastHeartbeatRecv" : ISODate("2022-07-20T09:26:18.010Z"),
+      "pingMs" : NumberLong(0),
+      "lastHeartbeatMessage" : "",
+      "syncingTo" : "pce-mongodb-replica-0.pce-mongodb:27017",
+      "syncSourceHost" : "pce-mongodb-replica-0.pce-mongodb:27017",
+      "syncSourceId" : 0,
+      "infoMessage" : "",
+      "configVersion" : 4
+    },
+    {
+      "_id" : 2,
+      "name" : "pce-mongodb-replica-2.pce-mongodb:27017",
+      "health" : 1,
+      "state" : 2,
+      "stateStr" : "SECONDARY",
+      "uptime" : 31,
+      "optime" : {
+        "ts" : Timestamp(1658309173, 1),
+        "t" : NumberLong(1)
+      },
+      "optimeDurable" : {
+        "ts" : Timestamp(1658309173, 1),
+        "t" : NumberLong(1)
+      },
+      "optimeDate" : ISODate("2022-07-20T09:26:13Z"),
+      "optimeDurableDate" : ISODate("2022-07-20T09:26:13Z"),
+      "lastHeartbeat" : ISODate("2022-07-20T09:26:18.007Z"),
+      "lastHeartbeatRecv" : ISODate("2022-07-20T09:26:18.536Z"),
+      "pingMs" : NumberLong(0),
+      "lastHeartbeatMessage" : "",
+      "syncingTo" : "pce-mongodb-replica-1.pce-mongodb:27017",
+      "syncSourceHost" : "pce-mongodb-replica-1.pce-mongodb:27017",
+      "syncSourceId" : 1,
+      "infoMessage" : "",
+      "configVersion" : 4
+    }
+  ],
+  "ok" : 1,
+  "$clusterTime" : {
+    "clusterTime" : Timestamp(1658309173, 1),
+    "signature" : {
+      "hash" : BinData(0,"AAAAAAAAAAAAAAAAAAAAAAAAAAA="),
+      "keyId" : NumberLong(0)
+    }
+  },
+  "operationTime" : Timestamp(1658309173, 1)
+}
+rs0:PRIMARY>
+```
+
+The **members** section of the status output shows three replicas. The pod *pce-mongodb-replica-0* is listed as the **PRIMARY** replica, while the other two pods, *pce-mongodb-replica-1* & *pce-mongodb-replica-2*, are listed as the **SECONDARY** replicas.
+
+The MongoDB replica set deployment is set up and ready to operate now. You can download and install the [MongoDB Compass](https://www.mongodb.com/docs/compass) and set up an external connection to connect to the MongoDB application deployed in the Kubernetes cluster. Using the powerful MongoDB Compass GUI, you can query, aggregate, and analyze the MongoDB data from the MongoDB deployment.
+
+## Scale MongoDB Application
+
+If you want to add another replica set to the MongoDB deployment to scale up the MongoDB application, you can run the following kubectl command:
+
+```markdown
+$ kubectl scale statefulset <name> -n <namespace> --replicas=<number of replicas>
+```
+
+Then you follow up the *step 1* in previous section to connect to the first MongoDB pod and repeat the *step 5* to add the new replica pod.
```

Comment:
Recommend rephasing this to "Then, follow up with *Step 1* from the previous section to connect to the first MongoDB pod and repeat *Step 5* to add the new replica pod."